Lc5 기능 설명 작성

sadie100 opened the pull request.

sadie100 committed:
lc5: lc5 기능 설명 작성

sadie100 committed:
Merge branch 'main' of https://github.com/bsgp/docs

gnseo commented on mkdocs.yml:
```diff
@@ -15,6 +15,10 @@ nav:
           - 'RFC Step 추가': 'eai/functions/add_rfc_step.md'
+    - LC5:
+      - Overview: 'lc5/index.md'
+      - Functions: 'lc5/functions/index.md'
+        
```
아래와 같은 구조로 문서를 만들어주세요.

```suggestion
      - "미리 정의된 컴포넌트 라이브러리": 'lc5/prebuilt_components.md'
      - "미리보기": 'lc5/preview.md'
      - "반응형 디자인 지원": "lc5/responsive.md"
      - "데이터 바인딩": "lc5/binding.md"
      - "템플릿 및 테마": "lc5/templates.md"
      - "버전 관리": "lc5/versioning.md"
```

gnseo reviewed (changes requested):
그리고 내용을 조금 더 자세하게, 풍부하게 작성하면 좋겠습니다.

sadie100 committed:
Update mkdocs.yml

Co-authored-by: GoonNahm Seo <gnseo@bsgglobal.com>

sadie100 committed:
chore(lc5) : 기능 리스트 파일 분리

sadie100 committed:
fix(lc5) : Overview 작성

sadie100 committed:
lc5: overview 내용 보완

sadie100 committed:
lc5: 미리 정의된 컴포넌트 페이지 내용 보완(이미지 추가, 이름 변경, 내용 보완 등)

sadie100 committed:
feat(lc5): 컴포넌트 설명 추가, 컴포넌트 이미지 추가

sadie100 committed:
build: yarn script 커맨드 추가

sadie100 committed:
chore(lc5) : lc5/Functions를 lc5/Concepts로 변경

sadie100 committed:
feat(lc5) : Builder, Meta, Renderer 문서 작성 & preview 문서 삭제(새로 생성된 문서에 통합함)

sadie100 committed:
fix(lc5): 미리 정의된 컴포넌트 내용 보완, 글 목록에서 위치 변경

sadie100 committed:
fix(lc5): 반응형 디자인 섹션을 미리 정의된 컴포넌트 섹션에 통합시키고 별도의 페이지로 분리되어 있던 것을 삭제함

sadie100 committed:
fix(lc5) : 미리 정의된 컴포넌트 파트의 내용 짜임새 다듬음

gnseo commented:
현상태를 draft로 변경했습니다. 완료되면 "Ready for review"를 클릭하면 될겁니다.
<img width="898" alt="Screenshot 2024-07-19 at 14 03 12" src="https://github.com/user-attachments/assets/b37721f7-e176-4249-acb1-9eaa961f6f37">

sadie100 committed:
feat(lc5) : Builder, meta, renderer에서 meta를 분리하고 Meta, State 문서를 따로 작성함

sadie100 committed:
feat(lc5) : 함수 파트 1차 작성

sadie100 committed:
feat(lc5) : quick start 페이지 작성

sadie100 committed:
fix(lc5): 퀵스타트 페이지 보완

sadie100 committed:
fix(lc5):  커스텀 함수 누락된 내용 작성, 초기함수->최초함수 용어 변경(log와 용어 맞춤)

sadie100 committed:
feat(lc5): Builder UI 세부사항 초안 작성(Component Dialog)

sadie100 committed:
fix(lc5) : 퀵스타트 문서 보완(IHub 관련 내용 추가)

sadie100 committed:
fix(lc5) : function 내용 변경 => 컴포넌트 디테일 페이지에 있던 async function 관련 내용을 함수 페이지로 이동시키고 해당 내용 중심으로 전면 수정함

sadie100 committed:
fix(lc5): 빌더 렌더러 페이지 내용 수정

sadie100 committed:
feat(lc5): concepts overview 작성

sadie100 committed:
feat(lc5) : 1. Reference 섹션을 따로 생성하고 Concepts에 있던 일부 구체적인 명세 관련 내용을 Reference로 이동시킴.
2. Template 삭제
3. 변경된 구조 및 추가된 파일에 맞게 mkdocs, index 수정
4. reference/overview 작성

sadie100 committed:
fix(lc5) : state 구조 페이지 소개글 수정

sadie100 committed:
feat(lc5): Builder UI 문서 작성

sadie100 committed:
feat(lc5) : 경로 및 버전관리 작성중

sadie100 committed:
fix(lc5) : Overview의 특징 부분 보완함

sadie100 committed:
fix(path_version) : 버전관리 작성 완료

sadie100 committed:
fix(lc5) : 문서 검토 및 기타 수정(개발자->사용자로 용어 통합, 문서 끝에 연재물 형식으로 추가한 문구 삭제, 제목 수정 등)

sadie100 commented:
팀장님 라이브러리 공식문서 스타일로 일단 써보긴 했는데 다 쓰고 나니 log랑 굉장히 비슷한 내용구성이 된것 같아 이게 의도하신 게 맞는가 싶은데요, 한번 리뷰해주시면 감사하겠습니다.

gnseo committed:
Merge branch 'main' into lc5

gnseo commented:
> 팀장님 라이브러리 공식문서 스타일로 일단 써보긴 했는데 다 쓰고 나니 log랑 굉장히 비슷한 내용구성이 된것 같아 이게 의도하신 게 맞는가 싶은데요, 한번 리뷰해주시면 감사하겠습니다.

PR에서는 비주얼을 확인할수 없어서 일단 배포하고 보겠습니다.
